src/Symfony/Bridge/Doctrine/Tests/Middleware/IdleConnection/DriverTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Tests\Middleware\IdleConnection;
+
+use Doctrine\DBAL\Driver as DriverInterface;
+use Doctrine\DBAL\Driver\Connection as ConnectionInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Doctrine\Middleware\IdleConnection\Driver;
+
+class DriverTest extends TestCase
+{
+    public function testConnect()
```

Comment:
```suggestion
    /**
     * @group time-sensitive
     */
    public function testConnect()
```

and also list the namespace in src/Symfony/Bridge/Doctrine/phpunit.xml.dist and in the root phpunit.xml.dist